runtime/lua/vim/filetype/detect.lua

```diff
@@ -1172,7 +1172,8 @@ function M.sh(path, contents, name)
       vim.b[b].is_kornshell = nil
       vim.b[b].is_sh = nil
     end
-  elseif matchregex(name, [[\<sh\>]]) then
+    -- Ubuntu links sh to dash
```

Comment:
I think this comment should be below the `elseif` and match the original comment in Vim better.

Reply:
But that's how the comments are ordered in the rest of this file. Being inconsistent is even more confusing.

Reply:
(And I don't think the Vim comment contains any more information; it is just longer. We port the logic, not the code.)